src/hooks/use-sync-states-progress-info.ts

```diff
@@ -73,17 +91,17 @@ export function useSyncStatesProgressInfo() {
 		return {
 			creatingBackup: {
 				key: 'creatingBackup',
-				progress: 30,
+				progress: 20,
 				message: __( 'Creating backup…' ),
 			},
 			uploading: {
 				key: 'uploading',
-				progress: 50,
+				progress: 40,
 				message: __( 'Uploading Studio site…' ),
 			},
 			importing: {
 				key: 'importing',
-				progress: 80,
+				progress: IMPORTING_INITIAL_VALUE,
```

Comment:
Similarly to the previous comment, This variable is being used also in the Pull process.